core/src/main/java/org/fao/geonet/resources/CMISResources.java

```diff
@@ -481,8 +481,22 @@ public void close() throws IOException {
                         try {
                             // If the document is found then we are updating the existing document.
                             Document doc = (Document) CMISConfiguration.getClient().getObjectByPath(key, oc);
-                            doc.updateProperties(properties, true);
-                            doc.setContentStream(contentStream, true, true);
+
+                            // If using major versioning then we have the option of making next version a minor or major.
```

Comment:
This code looks the same as in https://github.com/geonetwork/core-geonetwork/pull/5357/files#diff-574b3d4af408a3edd533174cd787c1a07a05eba3563e250cee33321ea5b93dafR189, maybe can be moved to an utility class like the `CMISUtils` in #5359

This change can be done as part of other PR when this one and #5359 are merged, but to consider.

Reply:
Once the 2 PR are merged, I plan on doing another PR to refactor the code and fix the configuration variables so that it uses spring placeholders instead of using the getenv which would be more inline with the other bean configurations.  I can look into adding the change in that PR.